src/app/examples/progress-bar-example/progress-bar-example.component.tns.html

```diff
@@ -0,0 +1,12 @@
+<ScrollView>
```

Comment:
There is some more indentation issues here 😄 

Sorry i missed that first time around.